### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Mantis #0796534
- This global variable is automatically get created by the MITRE connector during connector configuration and used for data ingestion under "MITRE Framework" SP
- Hence removed the global variable 'mitre_version' from this SP as there is no use for it

Mantis #0781959
Updated info.json, made the following changes
- Updated connector entries as per the latest format(removed connector version and updated connectors\data.json)
- Updated recordset entry to the latest format(added module name and apiName)
- removed discarded keys such as 'exported_from' and 'exported_by'

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -28,17 +28,15 @@
...
     ],
     "prerequisite": "* Ensure that SOC Simulator connector is configured\n* Ensure that MITRE ATT&CK Connector as well as data ingestion is configured",
-    "category": [],
     "certified": "true",
     "publisher": "Fortinet",
     "description": "The solution pack demonstrates users a variety of scenarios and use cases around threat hunting based on using the information provided by the MITRE ATT&CK Framework.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-mitre-attack-threat-hunting/blob/develop/README.md",
+    "category": [],
     "supportInfo": "support@fortinet.com",
-    "help": "https:\/\/github.com\/fortinet-fortisoar\/solution-pack-mitre-attack-threat-hunting\/blob\/develop\/README.md",
```
Comment (cs-suyog-jinturkar) [inline info.json:36]: Remove support@fortinrt.com

File: info.json [hunk 2]
```diff
@@ -28,17 +28,15 @@
         }
     ],
     "prerequisite": "* Ensure that SOC Simulator connector is configured\n* Ensure that MITRE ATT&CK Connector as well as data ingestion is configured",
-    "category": [],
     "certified": "true",
     "publisher": "Fortinet",
     "description": "The solution pack demonstrates users a variety of scenarios and use cases around threat hunting based on using the information provided by the MITRE ATT&CK Framework.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-mitre-attack-threat-hunting/blob/develop/README.md",
```
Comment (cs-suyog-jinturkar) [inline info.json:34]: Description should contain connector name as suggested by Amit

Comment (cs-rohit-khune) [reply info.json:34]: Added connector name in SP description

File: info.json (hunk 1 above)
Comment (cs-rohit-khune) [reply info.json:36]: replaced `support@fortinet.com` with `Fortinet Customer Support`

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**